pkg/spireentry/reconciler_test.go

```diff
@@ -75,3 +76,31 @@ func TestMakeEntryKey(t *testing.T) {
 		require.Equal(t, makeEntryKey(a), makeEntryKey(b))
 	})
 }
+
+func TestIsNamespaceIgnored(t *testing.T) {
+	tests := []struct {
+		ignoredNamespaces []string
+		namespace         string
+		expected          bool
+	}{
+		{[]string{"s([a-z]+)re", "default"}, "spire", true},
+		{[]string{"s([a-z]+)re", "default"}, "default", true},
+		{[]string{"s([a-z]+)re", "default"}, "spiffe", false},
+		{[]string{"s([a-z]+)re", "default"}, "kubernetes", false},
```

Comment:
It would be good to vary the regex strings

Reply:
nm the above. Lets keep the regex strings constant and vary just `namespace` and `expected`. Can you move the regex string above the test struct and compile it. Then go through the test cases. That should work better.